flake.nix

```diff
@@ -47,6 +47,8 @@
             cargo-udeps
             cargo-sort
             cmake
+
+            postgresql_15
```

Comment:
The default docker version is already 16 so wondering why we choose 15 here.

Reply:
15 is the newest version in nixpkgs. If you just do `postgresql` you get an even older version (14.9)

Reply:
There should be a _16, probably need to `nix flake update` if it's not there.

https://search.nixos.org/packages?channel=unstable&show=postgresql_16&from=0&size=50&sort=relevance&type=packages&query=postgresql_16

Reply:
Ah that did the trick